src/main/resources/fxml/oauth_controller.fxml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
```

Comment:
@tyle8552 Just a remind for both of us that we should check this is in all FXML files. I know some of mine don't have it.